index.bs

```diff
@@ -13616,6 +13623,189 @@ The [=remote end steps=] given |session| and |command parameters| are:
+        element: script.SharedReference,
+        ? card: autofill.Card,
+        ? address: autofill.Address
+      }
+
+      autofill.Field = {
+        name: autofill.FieldName,
+        value: text
+      }
+
+      autofill.Card = {
+        number: text,
+        name: text,
+        expiryMonth: text,
+        expiryYear: text,
+        cvc: text
+      }
+
+      autofill.Address = {
+        fields: [* autofill.Field]
+      }
+
+    </pre>
+   </dd>
+   <dt>Return Type</dt>
+   <dd>
+    <code>
+     EmptyResult
+    </code>
+   </dd>
+</dl>
+
+<div algorithm="remote end steps for autofill.Trigger">
+
+The [=remote end steps=] with |session| and |command parameters| are:
+
+1. Let |navigable id| be the value of the <code>context</code> field of
+   |command parameters|.
+
+1. Let |navigable| be the result of [=trying=] to [=get a navigable=]
+    with |navigable id|.
+
+1. Let |document| be |navigable|'s [=active document=].
+
+1. Let |environment settings| be the [=environment settings object=] whose
+   [=relevant global object=]'s <a>associated <code>Document</code></a> is
+   |document|.
+
+1. Let |realm| be |environment settings|' [=realm execution context=]'s
+  Realm component.
+
+1. Let |element id| be the value of the <code>element</code> field of
+   |command parameters|.
+
+1. Let |element| be the result of [=trying=] to [=deserialize remote reference=]
+   with the |element id|, |realm|, and |session|.
+
+1. If |element| is not a [=form-associated element=] or |element|'s [=form owner=] is
+   null, return an [=error=] with [=error code=] [=invalid element state=].
+
+1. Let |card| be the value of the <code>card</code> field of |command parameters| if present, or null otherwise.
+
+1. Let |address| be the value of the <code>address</code> field of |command parameters| if present, or null otherwise.
+
+1. If, for any reason, the remote end is unable to [=autofill=], return error with
+   [=error code=] [=unsupported operation=].
+
+1. If both |card| and |address| are not null, return error with [=error code=]
+   [=invalid argument=].
+
+1. If both |card| and |address| are null, return error with [=error code=]
+   [=invalid argument=].
+
+1. Let |parsed card| be the result of parsing |card| as an <code>autofill.Card</code>.
+
+1. Let |parsed address| be the result of parsing |card| as an <code>autofill.Address</code>.
+
+1. The [=/user agent=] should [=autofill=] |element| and |element|'s [=form
+   owner=], while taking into account the contents of |parsed card| and |parsed address|.
+
+1. Return [=success=] with data null.
+
+</div>
+
+#### The autofill.FieldName Type #### {#type-autofill-FieldName}
+
+<pre class="cddl" data-cddl-module="local-cddl">
+AutofillToken = "name" /
+                "honorific-prefix" /
+                "given-name" /
+                "additional-name" /
+                "family-name" /
+                "honorific-suffix" /
+                "nickname" /
```

Comment:
Is the intent here to be exhaustive? I feel like one-time-code, nickname, organization-title, bday, impp are not relevant if we are focusing on autofill for shopping cart checkout pages.